networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,39 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: add rhel latest repos
+    yum_repository:
+      name: rhel_latest
+      description: taken form https://github.com/redhat-performance/perf-dept/blob/master/repo_files/rhel7-latest.repo
+      baseurl: http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/
+
+  - name: install reqiured pkgs
+    yum: state=present name={{ item }}
```

Comment:
Please stay consistent in ansible command formatting (compressed vs multi line), in this case expand.